hugegraph-test/src/main/java/com/baidu/hugegraph/core/PropertyCoreTest.java

```diff
@@ -462,13 +467,13 @@ public void testTypeFloat() {
                                              1f, 3f, 3f, 127f, 128f));
 
         Assert.assertThrows(IllegalArgumentException.class, () -> {
-            property("float", Double.MAX_VALUE);
+            property("float", 'a');
```

Comment:
what's the behavior of property("float", Double.MAX_VALUE)? can also test it

Reply:
treat as `Infinity`. add a assertEquals for it